src/parser/parsers/items.py

```diff
@@ -104,7 +104,7 @@ def _parse_item(self, key):
         return parsed_item_data
 
     def _is_disabled(self, item):
-        is_disabled = False
+        is_disabled = True
```

Comment:
Why is this now defaulted as `True`?

Reply:
I only see like 15 items with `"IsDisabled": false`, so I think defaulting to False is correct here